plonky3/Cargo.toml

```diff
@@ -13,31 +13,32 @@ rand = "0.8.5"
 powdr-analysis = { path = "../analysis" }
 powdr-executor = { path = "../executor" }
 
-p3-air = { git = "https://github.com/powdr-labs/Plonky3.git", rev = "6afe4f75" }
-p3-matrix = { git = "https://github.com/powdr-labs/Plonky3.git", rev = "6afe4f75" }
-p3-field = { git = "https://github.com/powdr-labs/Plonky3.git", rev = "6afe4f75" }
-p3-uni-stark = { git = "https://github.com/powdr-labs/Plonky3.git", rev = "6afe4f75" }
-p3-commit = { git = "https://github.com/powdr-labs/Plonky3.git", rev = "6afe4f75", features = [
-  "test-utils",
+p3-air = { git = "https://github.com/powdr-labs/Plonky3.git", rev = "b38d3fa" }
```

Comment:
what's in this commit?

Reply:
it's the latest on main but CI doesn't pick it up somehow, see description